src/main/resources/hudson/plugins/release/dashboard/RecentReleasesPortlet/main.jelly

```diff
@@ -61,7 +61,7 @@ THE SOFTWARE.
 	        </td>
 	        <td>
 	          <a href="${b.url}console">
-	            <img src="${imagesURL}/${subIconSize}/terminal.gif" title="${%Console output}" alt="${%Console output}" border="0" />
+				<l:icon class="icon-sm" src="symbol-terminal" tooltip="${%Console output}" />
```

Comment:
![Screenshot 2024-01-04 at 10 38 49](https://github.com/jenkinsci/release-plugin/assets/13383509/1215a943-2ed0-43cf-802b-8e90d14497df)
No idea where GH takes the tabs from, looks fine in the IDE 🤷 
![Screenshot 2024-01-04 at 10 38 55](https://github.com/jenkinsci/release-plugin/assets/13383509/45f558a4-31ee-4fd3-94d8-9ea1b229c814)

Reply:
your ide will be hiding it, if you reformat the file it would fix it but not worried for this project.